modules/ROOT/pages/search-cms.adoc

```diff
@@ -0,0 +1,59 @@
+= Optionally Configure Search in CMS Content
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images/
+
+Anypoint API Community Manager stores some content, such as news articles and blog posts, in a content management system or _CMS_. To include this content in search results, enable the Salesforce CMS content manager.
+
+== CMS Application Permissions
+
+Open your administrator profile and ensure that it has the following CMS application permissions:
+
+* *Manage Content Permissions*: Create, edit, and delete library permissions in Salesforce CRM Content.
+* *Manage Content Properties*: Create, edit, and delete custom fields in Salesforce CRM Content.
+* *Manage Record Types and Layouts for Files*: Create, edit, and delete content types in Salesforce CRM Content.
+* *Manage Salesforce CRM Content*: Create, edit, and delete libraries and library memberships.
+
+== Enable Salesforce CMS
+
+To enable Salesforce CMS for API Community Manager:
+
+. In the administrative interface, open CMS Administrator.
+. If the interface shows a message that you have no content available yet, create content:
+.. Open the control panel.
+.. Click the apps icon in the top left and use the `Search apps and items...` box to find and open the Salesforce CMS app.
+.. Select *CMS Workspaces*.
+.. Click *Add Workspace*.
+.. Enter the name and description of the workspace.
+.. Click *Next*.
+.. Select the channel for each community that has CMS content to include in your community's search results.
+.. If you have contributors, add them.
+.. Choose the languages for your translated content.
+.. Click *Next*.
+.. Verify the information.
+.. Click *Done*.
+.. Create a new collection and upload content.
+. Enable search in the community channel:
+.. Open the control panel.
+.. Click the apps icon in the top left and use the `Search apps and items...` box to find and open the Salesforce CMS app.
+.. Open the *CMS Home* tab and use the drop-down menu to open *CMS Channels*.
+.. Find your site on the list and, in the *Search* column, open the drop-down menu and click *Edit*.
+.. In the *Channel Settings* menu, open the *Search* tab and click *Enable Search*.
+.. In the *Search By Content Type* menu, select which content types are enabled and disabled for search.
+.. Click *Save*.
+. Add your community URL to remote site settings:
+.. Open *Setup*.
+.. Use the `Quick Find` box to find and open *Remote Site Settings*.
```

Comment:
If the box is labelled Quick Find, it should be in bold: *Quick Find*
monotype `` is for values provided by the computer or by the user, that are not permanently in the UI

Reply:
Yes, this label is weird because it appears inside the box and disappears when the user selects the box, but as I think about your point, I conclude that it is a label and it should be bold. Fixed.